oneflow/user/ops/offset_to_ndindex_op.cpp

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright 2020 The OneFlow Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+#include "oneflow/core/framework/framework.h"
+
+namespace oneflow {
+
+namespace user_op {
+
+REGISTER_USER_OP("offset_to_ndindex")
+    .Input("offset")
+    .Input("dims")
+    .Output("out")
+    .SetTensorDescInferFn([](user_op::InferContext* ctx) -> Maybe<void> {
+      const TensorDesc* offset = ctx->TensorDesc4ArgNameAndIndex("offset", 0);
+      const TensorDesc* dims = ctx->TensorDesc4ArgNameAndIndex("dims", 0);
+      int64_t offset_num_axes = offset->shape().NumAxes();
+      int64_t dims_num_axes = dims->shape().NumAxes();
+      CHECK_GT_OR_RETURN(offset_num_axes, 0);
+      CHECK_GT_OR_RETURN(dims_num_axes, 0);
```

Comment:
```suggestion
      CHECK_GT_OR_RETURN(dims->shape().NumAxes(), 0);
```
这两处的局部变量除了检查外，没其它地方用到，可以省掉